content/akeneo-web-api.yaml

```diff
@@ -2355,6 +2354,46 @@ paths:
           $ref: "#/responses/415Error"
         422:
           $ref: "#/responses/422Error"
+    patch:
+      summary: Update/create several association types
+      operationId: "several_association_types_patch"
+      tags:
+        - Association types
+      description: This endpoint allows you to update and/or create several association types at once.
+      x-body-by-line: Contains several lines, each line is an association type in JSON standard format
+      parameters:
+        - name: body
+          in: body
+          schema:
+            $ref: "#/definitions/AssociationType"
+            x-examples:
+              x-example-1:
+                '{"code":"NEW_SELL"}'
```

Comment:
I would not use uppercase for these codes as we do not use it in the other examples. Even if it is valid (and this is what we have in our demo data), we should keep things uniform.